### PR TITLE
[IMP] subscriptions: update renewals doc

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3667181&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

**Comment by ksc-odoo** [comment]
Alrighty @meng-odoo this one is ready for its first round of Peer Review 👍

File: content/applications/sales/subscriptions/renewals.rst [hunk 1]
```diff
@@ -1,80 +1,70 @@
...
-but, in some cases, manual subscription renewals are still the preferred option.
+Most companies that offer subscriptions prefer to automate the renewals process for customers, but,
+in some cases, manual subscription renewals are still used.
 
-With **Odoo Subscriptions**, you can have all your subscriptions in one application, suggest an
-automatic subscription renewal to your customers (as well as a manual one) and, finally, filter all
-your subscriptions and easily find those to renew (with the help of the tag *To renew*).
+With the Odoo *Subscriptions* application, a company can accurately manage all subscription renewals
+in a single platform. Users can even filter all current subscriptions to quickly locate customers
+who need to renew.
```
Comment (StraubCreative) [inline content/applications/sales/subscriptions/renewals.rst:20]: I think the previous instance of this paragraph was better in that it listed out, in a general way, what Odoo Subscriptions could do in regard to renewals. It didn't qualify with adjectives or pitch features.

Here is my suggestion to return back to that more neutral tone:

```suggestion
With the Odoo *Subscriptions* application, a company can manage all of their subscriptions in one
place. Renewals can be processed automatically or manually, include additional products or upsells
per renewal order, and be filtered in batch views to quickly locate customers who need to renew
their subscriptions.
```

File: content/applications/sales/subscriptions/renewals.rst [hunk 2]
```diff
@@ -1,80 +1,70 @@
...
+=====================
 
-Before renewing a subscription, be sure to check out our documentation on how to
-:doc:`Create a quotation <../subscriptions>` using subscription
-products. Indeed, once confirmed, a quotation becomes a sales order and a new subscription is
-automatically created. Therefore, this subscription has the status *In progress*. From there, you
-have the possibility to renew the subscription. In the Other Info tab, underneath the To Renew
-section, you can activate the *To renew* option. When activated, a yellow tag automatically appears
-in the upper right corner of the subscription.
+In order to renew a subscription, a quotation with a subscription product(s)/service(s) **must** be
```
Comment (StraubCreative) [inline content/applications/sales/subscriptions/renewals.rst:25]: List the necessary requirement only:
- Only a singular product is required
- A subscription service is still a product (albeit a digital + recurring one)

```suggestion
In order to renew a subscription, a quotation with a subscription product **must** be
```

File: content/applications/sales/subscriptions/renewals.rst [hunk 3]
```diff
@@ -1,80 +1,70 @@
...
+.. image:: renewals/renew-button.png
+  :align: center
+  :alt: Renew button on subscription sales order with Odoo Subscriptions.
 
-When a subscription needs to be renewed, you have the possibility to use a new button called
-*Renewal quotation*. By clicking on it, a new quotation is created. From there, start a basic sales
-flow allowing you to send the quotation by email to your customers or to confirm it. It is better to
-first *Send by email* the quotation to your customers in order to have their confirmation and, then,
-*Confirm* it in **Odoo Sales**.
+When the :guilabel:`Renew` is clicked, Odoo instantly presents a new renewal quotation, complete
```
Comment (StraubCreative) [inline content/applications/sales/subscriptions/renewals.rst:39]:
```suggestion
When the :guilabel:`Renew` button is clicked, Odoo instantly presents a new renewal quotation, complete
```

File: content/applications/sales/subscriptions/renewals.rst [hunk 4]
```diff
@@ -1,80 +1,70 @@
...
-   from the previous subscription". Once confirmed by your customers, this quotation becomes a
-   sales order and a new sale is mentioned in the upper right corner of the subscription.
+.. image:: renewals/renewal-quotation.png
+  :align: center
+  :alt: Renewal quotation in the Odoo Subscriptions application.
 
-   .. image:: renewals/renew-a-quotation.png
-     :align: center
-     :alt: Renew a quotation with Odoo Subscriptions
+From here, a standard sales flow can occur, which is typically started by clicking :guilabel:`Send
```
Comment (StraubCreative) [inline content/applications/sales/subscriptions/renewals.rst:46]: optional suggestion around specificity
```suggestion
From here, a standard sales flow can occur in order to confirm the quotation. This is typically begins by clicking :guilabel:`Send
```

File: content/applications/sales/subscriptions/renewals.rst [hunk 5]
```diff
@@ -1,80 +1,70 @@
...
+by Email`. This sends a copy of the quotation to the customer for them to confirm, and eventually,
```
Comment (StraubCreative) [inline content/applications/sales/subscriptions/renewals.rst:47]:
```suggestion
by Email`. This sends a copy of the quotation to the customer, by email, for them to confirm, and eventually,
```

File: content/applications/sales/subscriptions/renewals.rst [hunk 6]
```diff
@@ -1,80 +1,70 @@
...
-to your *Subscriptions dashboard* and use the filter *To renew*.
+When that :guilabel:`Sales History` smart button is clicked, Odoo reveals a separate page,
+showcasing the different sales orders attached to this subscription, along with their individual
+:guilabel:`Subscription Status`.
 
-.. image:: renewals/subscriptions-dashboard-with-the-to-renew-filter.png
+.. image:: renewals/sales-history-page.png
   :align: center
-  :alt: List view of all subscriptions and use of the filter to renew in Odoo Subscriptions
+  :alt: Renewal quotation in the Odoo Subscriptions application.
```
Comment (StraubCreative) [inline content/applications/sales/subscriptions/renewals.rst:67]: I believe once a renewal is confirmed the **MRR** smartbutton becomes available too. Can we talk about this?

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @ksc-odoo 
Good rewrite.
Some quick changes to improve further, see below.

**Comment by ksc-odoo** [comment]
@StraubCreative Hey There! 👋 Just finished making all your suggested edits and added a portion at the end to briefly discuss the MRR button (with an additional screenshot, as well). Please feel free to give this another quick review whenever you get a chance. If there's anything else that needs to be modified/adjusted, just let me know. Thanks again for all the super-helpful feedback 👍

**Review by StraubCreative — APPROVED** [review]
Looks good @ksc-odoo 

One item for your consideration: most of the screenshots are a bit small and hard to read. Zooming in on compressed images unfortunately doesn't help too much here, so maybe expand the widths a bit to the tablet breakpoint of 768px or low res desktop 933-1024px, if needed.

Up to you though, merge when ready 👍 

@robodoo delegate=ksc-odoo

**Pull request closed without merging**